airflow/requirements.txt

```diff
@@ -17,3 +17,5 @@ structlog==21.5.0
 typer==0.4.0
 humanize==3.14.0
 backoff==1.11.1
+pydantic==1.9.0
+slugify==0.0.1
```

Comment:
I'd recommend https://pypi.org/project/python-slugify/ seems to be better supported/active

Reply:
We ended up having to drop this dependency entirely because it introduced too many conflicts & made the docker image un-buildable.